src/js/components/projects/detail.tsx

```diff
@@ -45,112 +43,61 @@ const ProjectDetail = (props: RouteComponentProps) => {
   // This redundant check is used to satisfy TypeScript...
   /* istanbul ignore if */
   if (!product || !project) {
-    return <ProductNotFound />;
+    return <FourOhFour />;
   }
 
   if (
     (productSlug && productSlug !== product.slug) ||
     (projectSlug && projectSlug !== project.slug)
   ) {
-    // Redirect to most recent product slug
+    // Redirect to most recent product/project slug
     return <Redirect to={routes.project_detail(product.slug, project.slug)} />;
   }
 
-  const projectDescriptionHasTitle =
-    project.description &&
-    (project.description.startsWith('<h1>') ||
-      project.description.startsWith('<h2>'));
-
   return (
     <DocumentTitle
       title={`${project.name} | ${product.name} | ${i18n.t('MetaShare')}`}
     >
-      <>
-        <PageHeader
-          className="page-header slds-p-around_x-large"
-          title={project.name}
-          info={<RepoLink url={product.repo_url} shortenGithub />}
+      <DetailPageLayout
+        title={project.name}
+        description={project.description}
+        repoUrl={product.repo_url}
+        breadcrumb={[
+          { name: product.name, url: routes.product_detail(product.slug) },
+          { name: project.name },
+        ]}
+      >
+        <Button
+          label={i18n.t('Submit Project')}
+          className="slds-size_full slds-m-bottom_x-large"
+          variant="outline-brand"
+          disabled
         />
-        <div
-          className="slds-p-horizontal_x-large
-            slds-p-top_x-small
-            ms-breadcrumb"
-        >
-          <BreadCrumb
-            trail={[
-              <Link to={routes.home()} key="home">
-                {i18n.t('Home')}
-              </Link>,
-              <Link to={routes.product_detail(product.slug)} key="home">
-                {product.name}
-              </Link>,
-              <div className="slds-p-horizontal_x-small" key={project.slug}>
-                {project.name}
-              </div>,
-            ]}
-          />
-        </div>
-        <div
-          className="slds-p-around_x-large
-            slds-grid
-            slds-gutters
-            slds-wrap"
-        >
-          <div
-            className="slds-col
-              slds-size_1-of-1
-              slds-medium-size_2-of-3
-              slds-p-bottom_x-large"
-          >
-            <Button
-              label={i18n.t('Submit Project')}
-              className="slds-size_full slds-m-bottom_x-large"
-              variant="outline-brand"
+        {tasks ? (
+          <>
+            <h2 className="slds-text-heading_medium slds-p-bottom_medium">
+              {tasks.length ? (
+                <>
+                  {i18n.t('Tasks for')} {project.name}
+                </>
+              ) : (
+                <>
+                  {i18n.t('Add a Task for')} {project.name}
+                </>
```

Comment:
Does this all need to be contained in a null element? Or can we just inline the strings?

Reply:
Because it's concatenating strings, there are two choices:

```js
`${i18n.t('Add a Task for')} ${project.name}`
```

or

```js
<>{i18n.t('Add a Task for')} {project.name}</>
```

I don't really have an opinion as to which is cleaner. 🤷‍♂

Reply:
Nor I. Just curious.